frontend/src/components/SocialMediaButton/index.js

```diff
@@ -1,59 +1,42 @@
+import { PropTypes } from "prop-types";
 import styles from "./index.module.css";
+import * as constants from "constants/Subvisual.js";
 
-import Github from "assets/imgs/socialButtons/github.png";
-import LinkedIn from "assets/imgs/socialButtons/linkedin.png";
-import Behance from "assets/imgs/socialButtons/behance.png";
-import Facebook from "assets/imgs/socialButtons/facebook.png";
-import Instagram from "assets/imgs/socialButtons/instagram.png";
-import Dribbble from "assets/imgs/socialButtons/dribble.png";
-import Medium from "assets/imgs/socialButtons/medium.png";
-import Twitter from "assets/imgs/socialButtons/twitter.png";
-
-const socialMediaButton = ({ name, username }) => {
+const SocialMediaButton = ({ name }) => {
   let logo = "";
   let link = "";
-  const links = {
-    github: "https://github.com/",
-    linkedin: "https://www.linkedin.com/company/", // if its an user: "https://www.linkedin.com/in/",
-    behance: "https://www.behance.net/",
-    facebook: "https://www.facebook.com/",
-    instagram: "https://www.instagram.com/",
-    dribbble: "https://dribbble.com/",
-    medium: "https://medium.com/",
-    twitter: "https://twitter.com/",
-  };
 
   if (name === "github") {
-    logo = Github;
-    link = links.github + username;
+    logo = constants.LOGOS.GITHUB;
+    link = constants.GITHUB;
   }
   if (name === "linkedin") {
-    logo = LinkedIn;
-    link = links.linkedin + username;
+    logo = constants.LOGOS.LINKEDIN;
+    link = constants.LINKEDIN;
   }
   if (name === "behance") {
-    logo = Behance;
-    link = links.behance + username;
+    logo = constants.LOGOS.BEHANCE;
+    link = constants.BEHANCE;
   }
   if (name === "facebook") {
-    logo = Facebook;
-    link = links.facebook + username;
+    logo = constants.LOGOS.FACEBOOK;
+    link = constants.FACEBOOK;
   }
   if (name === "instagram") {
-    logo = Instagram;
-    link = links.instagram + username;
+    logo = constants.LOGOS.INSTAGRAM;
+    link = constants.INSTAGRAM;
   }
   if (name === "dribbble") {
-    logo = Dribbble;
-    link = links.dribbble + username;
+    logo = constants.LOGOS.DRIBBBLE;
+    link = constants.DRIBBBLE;
   }
   if (name === "medium") {
-    logo = Medium;
-    link = links.medium + username;
+    logo = constants.LOGOS.MEDIUM;
+    link = constants.MEDIUM;
   }
   if (name === "twitter") {
-    logo = Twitter;
-    link = links.twitter + username;
+    logo = constants.LOGOS.Twitter;
+    link = constants.TWITTER;
   }
```

Comment:
Let's fix the logic of this in this PR too. Normally, I wouldn't suggest this, but it's a bit messy since this has all already been pushed up.

Let's create a function for this.

Reply:
I already fixed it on the other PR